.github/workflows/ci.yml

```diff
@@ -7,3 +7,44 @@ on:
     branches:
     - main
     - development
+  push:
+    branches:
+      - mkdocstrings
+jobs:
+  tox:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ['3.10']
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v4 # Use v4 for compatibility with pyproject.toml
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: pip
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install .
+          pip install tox
+
+      - name: Run tox
+        run: tox -e py
```

Comment:
-e py runs tox using any python version your runner has set up. if you don't specify that, it would test all the python versions set up in the tox.ini, but that would not take advantage of the parallelization of the github action